docs/en/enterprise-edition/rn/known-issues/known-fixed-issues.adoc

```diff
@@ -27,7 +27,7 @@ The list of fixed issues are not cumulative; only the issues that are fixed with
 |*RLP-133444*
 //Added post 24.2.2 when the issues was discovered. Moved to Fixed Issues in 3.1 preserving the details of the Known Issue.
 
-|The *Resource Tag Filter* option was incorrectly enabled for all users on the Prisma Cloud console. Ennablement should have been restricted to users who signed up for a Limited General Availability (LGA) trial.
+|The *Resource Tag Filter* option was incorrectly enabled for all users on the Prisma Cloud console. Ennablement is restricted to users who signed up for the Limited General Availability (LGA) trial.
```

Comment:
The filter is called "Asset Tag" since Darwin release, we should use that terminology here.

@jenjoe22